components/radio/stories/radio.stories.js

```diff
@@ -91,31 +95,132 @@ export default {
 	},
 };
 
-export const Default = (args) => html`
+// ===== Story specific templates ======
+
+// @todo Radio needs more robust coverage of all states, including hover, focus, etc.
+const ChromaticGroupTemplate = (args) => html`
 	<div style="display: flex; flex-direction: column; align-items: flex-start;">
 		${Template({
 			...args,
-			label: "Default"
+			label: "Default, not selected",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-1",
 		})}
 		${Template({
 			...args,
-			isEmphasized: true,
 			isChecked: true,
-			label: "Emphasized radio button label that is so long it has to wrap",
+			isEmphasized: false,
+			label: "Default, selected",
 			customStyles: {
 				"max-width": "220px",
-			}
+			},
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-2",
+		})}
+		${Template({
+			...args,
+			isChecked: true,
+			isEmphasized: true,
+			label: "Emphasized, selected",
+			customStyles: {
+				"max-width": "220px",
+			},
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-3",
 		})}
 		${Template({
 			...args,
 			isDisabled: true,
-			label: "Disabled"
+			label: "Disabled",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-4",
 		})}
 		${Template({
 			...args,
 			isDisabled: true,
 			isReadOnly: true,
-			label: "Read only"
+			label: "Read-only",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-5",
 		})}
 	</div>
 `;
+
+const BasicGroupTemplate = (args) => html`
+	<div style="display: flex; flex-direction: column; align-items: flex-start;">
+		${Template({
+			...args,
+			label: "Example label",
+			id: "radio-1-" + args?.id ?? "default",
+			name: "radio-example-" + args?.name ?? "default",
+		})}
+		${Template({
+			...args,
+			isChecked: true,
+			label: "Inititally selected radio button that has wrapping label text",
+			customStyles: {
+				"max-width": "220px",
+			},
+			id: "radio-2-" + args?.id ?? "default",
+			name: "radio-example-" + args?.name ?? "default",
+		})}
+	</div>
+`;
+
+// ===== Stories ======
+
+export const Default = (args) => {
+	return window.isChromatic() ? ChromaticGroupTemplate(args) : BasicGroupTemplate(args);
+};
+Default.tags = ["!autodocs"];
+
+export const Standard = BasicGroupTemplate.bind({});
+Standard.args = {
+	name: "standard",
+};
+Standard.tags = ["!dev"];
+
+export const Sizing = BasicGroupTemplate.bind({});
+Sizing.args = {
+	name: "sizing",
+};
+Sizing.tags = ["!dev"];
+Sizing.decorators = [withSizingWrapper];
```

Comment:
Do either of these stories need `chromatic: disabledSnapshot: true`? I could see an argument for both remaining unchanged and keeping their snapshots enabled, since the chromatic group doesn't have sizes in it, or an example of wrapping text. Just curious.

Reply:
After rebasing with Modes, I decided to disable the snapshots on these for now, and have added a "todo" about implementing the Variants() utility function with increased coverage (similar to what we saw was needed for Checkbox).